src/utils.js

```diff
@@ -69,6 +69,27 @@ export const setQueryParam = (param, value) => {
   localStorage.savedQueryParams = qs.toString()
 }
 
+export const appendQueryParam = (param, value) => {
+  const qs = new URLSearchParams(window.location.search)
+  if (qs.get(param)) {
+    qs.set(param, [...new Set([...qs.get(param).split(','), value])].join(',')) // remove duplication
+  } else {
+    qs.set(param, value)
+  }
+  window.history.replaceState(null, '', '?' + qs.toString())
+  localStorage.savedQueryParams = qs.toString()
+}
+
+// delete `value` entry from the specified parameter if it exists
+export const deleteQueryParam = (param, value) => {
```

Comment:
This name sounds like it's deleting the entire query parameter, not an element of a comma separated list. Could we name it `popQueryParamElement` and `appendQueryParamElement` or similar?

Likewise, could we rewrite it using `setQueryParam`? eg (untested):
```js
export const popQueryParamElement = (param, value) => {
  const existing = new Set[...(qs.get(param) ?? []).split(',')])
  existing.delete(value)
  setQueryParam(param, [...existing].join(','))
}
```